src/token/Cargo.toml

```diff
@@ -18,8 +18,7 @@ serde = "1.0"
 ic-canister = { git = "https://github.com/infinity-swap/canister-sdk", package = "ic-canister", tag = "v0.3.8" }
 ic-helpers = { git = "https://github.com/infinity-swap/canister-sdk", package = "ic-helpers", tag = "v0.3.8" }
 ic-storage = { git = "https://github.com/infinity-swap/canister-sdk", package = "ic-storage", tag = "v0.3.8" }
-# stable-structures = { git = "https://github.com/aewc/ic" }
-stable-structures = { path = "/Users/flyq/workspace/github-aewc/ic/rs/stable-structures"}
+stable-structures = { git = "https://github.com/aewc/ic" }
```

Comment:
hey @aewc , can this be modified to point to infinityswap's fork of the ic? https://github.com/infinity-swap/ic

Reply:
Yes, and I can add the the change of stable-structures to infinityswap's ic

Reply:
As the stable-structures is [not stable](https://github.com/dfinity/ic/commit/1c2d61213fff0c6f00c0497235a87da9d8b00eac) now, I want to not change the stable-structures directly, and just add a  new lib called stable memory in canister-sdk, the stable memory will export the stable-structures and add the functions to call the ic0.stable64_xxxx(), based on this, I need not fetch upstream often.